addons/binding/org.openhab.binding.zoneminder/src/main/java/org/openhab/binding/zoneminder/handler/ZoneMinderThingMonitorHandler.java

```diff
@@ -185,112 +242,193 @@ public void handleCommand(ChannelUID channelUID, Command command) {
+                                    getLogIdentifier(), command, channelUID.getId());
+
+                            if ((command == OnOffType.OFF) || (command == OnOffType.ON)) {
+                                dataConverter.setMonitorForceAlarmInternal((command == OnOffType.ON) ? true : false);
+                                String eventText = getConfigValueAsString(
+                                        ZoneMinderConstants.PARAMETER_MONITOR_EVENTTEXT);
+
+                                BigDecimal eventTimeout = getConfigValueAsBigDecimal(
+                                        ZoneMinderConstants.PARAMETER_MONITOR_TRIGGER_TIMEOUT);
+
+                                try {
+                                    connection = aquireSession();
+                                    if (connection == null) {
+                                        logger.error(
+                                                "{}: context='handleCommand' tags='ForceAlarm' - Command='{}' failed to obtain session",
+                                                getLogIdentifier(), command);
+                                        return;
+                                    }
+                                    monitorProxy = ZoneMinderFactory.getMonitorProxy(connection, getZoneMinderId());
+
+                                    if (command == OnOffType.ON) {
+                                        logger.info("{}: Activate 'ForceAlarm' to '{}' (Reason='{}', Timeout='{}')",
+                                                getLogIdentifier(), command, eventText, eventTimeout.intValue());
+
+                                        getZoneMinderBridgeHandler().activateForceAlarm(getZoneMinderId(), 255,
+                                                ZoneMinderConstants.MONITOR_EVENT_OPENHAB, eventText, "",
+                                                eventTimeout.intValue());
+
+                                        dataConverter.setMonitorForceAlarmInternal(true);
+
+                                        // Force a refresh
+                                        startAlarmRefresh(eventTimeout.intValue());
+
+                                    }
+
+                                    else if (command == OnOffType.OFF) {
+                                        logger.debug("{}: Cancel 'ForceAlarm'", getLogIdentifier());
+
+                                        getZoneMinderBridgeHandler().deactivateForceAlarm(getZoneMinderId());
+                                        dataConverter.setMonitorForceAlarmInternal(false);
+                                        // Stop Alarm Refresh
+                                        forceStopAlarmRefresh();
+
+                                    }
+                                    fetchMonitorGeneralData(monitorProxy);
+
+                                } catch (Exception ex) {
```

Comment:
Please catch specific exceptions instead of catching all exceptions. That way it is more clear what exceptions need to be handled and how this is done.